packages/esm-ward-app/src/beds/unassigned-patient.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import WardPatientCard from '../ward-patient-card/ward-patient-card';
+import styles from './unassigned-patient.scss';
+import { type WardPatient } from '../types';
+
+export interface UnassignedPatientProps {
+  wardPatient: WardPatient;
+}
+
+const UnassignedPatient: React.FC<UnassignedPatientProps> = ({ wardPatient }) => {
```

Comment:
FWIW, "UnassignedPatient" seemed a better name for this component than "UnassignedBed".  It seems like it should still be grouped with the "EmptyBed" and "OccupiedBed" components, but though it does make it seem a little weird to be in the "beds" directory.

Reply:
I would think that `UnassignedBed` = `EmptyBed`.

Reply:
Right, agreed, I was just saying that the ticket says to call this element "UnassignedBed", which seemed wrong to me because as you said UnassignedBed implies EmptyBed to me, so I went with UnassignedPatient.